examples/nanocoap_server/coap_handler.c

```diff
@@ -108,6 +146,7 @@ ssize_t _sha256_handler(coap_pkt_t* pkt, uint8_t *buf, size_t len, void *context
 const coap_resource_t coap_resources[] = {
     COAP_WELL_KNOWN_CORE_DEFAULT_HANDLER,
     { "/riot/board", COAP_GET, _riot_board_handler, NULL },
+    { "/riot/description", COAP_GET, _riot_block2_handler, NULL },
```

Comment:
_riot_block2_handler() plus the resource added to coap_resources adds as much text to the nanocoap_server executable as all of the block2 functions added to nanocoap.c.

I can't really argue with the implementation of _riot_block2_handler() as an example -- except that "/riot/description" is too long for a CoAP path. So shortened it to "/riot/ver" in [d80d15b](https://github.com/bergzand/RIOT/pull/4/commits/d80d15b6bdef85464121716b96c74989783f5343).

Reply:
I agree with the shorter path